lib/101/text_field_learn.dart

```diff
@@ -1,8 +1,12 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:flutter_full_learn/product/language/language_items.dart';
 
 class TextFieldLearn extends StatefulWidget {
-  const TextFieldLearn({super.key});
+  TextFieldLearn({super.key});
+
+  FocusNode focusNodeTextFieldOne = FocusNode();
+  FocusNode focusNodeTextFieldTwo = FocusNode();
```

Comment:
This kind of usage would cause memory leak. Init the focusNodes directly in states and dispose them properly.

```suggestion
class _TextFieldLearnState extends State<TextFieldLearn> {
  late final FocusNode focusNodeTextFieldOne;
  late final FocusNode focusNodeTextFieldTwo;
  
  @override
  void initState() {
    focusNodeTextFieldOne = FocusNode();
    focusNodeTextFieldTwo = FocusNode();
  }
  
  @override
  void dispose() {
    focusNodeTextFieldOne.dispose();
    focusNodeTextFieldTwo.dispose();
  }
```

Reply:
Done.